src/style.css

```diff
@@ -0,0 +1,61 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.container {
+  margin-top: 20px;
+}
```

Comment:
it's pretty much redundant
as well as the the wrapper itself in HTML - what's the purpose you're following with it?
```suggestion
```